### DISCUSSION TITLE
Github for startups

### DISCUSSION THREAD

**Discussion by yashjsalian23** [community/community#62268] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hi, my startup is applying for the GitHub for Startups program. I have one query: if I create an organization and then apply for the GitHub for Startup program, will this cause any issues, or will my request be denied since I had already created an org?

**Comment by AMohamedAakhil** [comment, marked as answer]
No it will not cause any issues. You can create an organization and then apply for the Github for Startups program.

**Reply by yashjsalian23** [reply]
Got it!,Thank you
